README.md

```diff
@@ -1,742 +1,119 @@
+| [IAM role for aws-ebs-csi-driver](modules/iam-for-ebs-csi-driver)                     | `-`       | Terraform module to manage IAM role used by EBS CSI driver                                                             |
 
-...
+Click on each package to see its full documentation.
 
-TBD
-
-## prerequisites
-
-there programs needs to be installed locally and available in `$PATH`
-
-- install [furyctl](https://github.com/sighup-io/furyctl#install)
-- install awscli
-- install [furyagent](https://github.com/sighup-io/furyagent#install)
-- install ansible
-- install kustomize v1 (v2 is incompatible at the moment)
-- install terraform
-- install openvpn
-- install git
-- install git-crypt
-
----
-
-## step-by-step cluster setup instructions
-
-Create a new empty folder for the project and _cd_ into it
+## Compatibility
 
-run `mkdir [project_name]`
+| Kubernetes Version |   Compatibility    | Notes                                               |
+|--------------------|:------------------:|-----------------------------------------------------|
+| `1.21.x`           | :white_check_mark: | No known issues                                     |
+| `1.22.x`           | :white_check_mark: | No known issues                                     |
+| `1.23.x`           | :white_check_mark: | No known issues                                     |
 
-> While working with infrastructure as code, it is highly encouraged to keep your infrastructure definition files under strict version control. This task is out of the scope of this tutorial, but you should probably want to do `git init` right now and `git commit` at all the checkpoints ;)
+Check the [compatibility matrix][compatibility-matrix] for additional informations about previous releases of the modules.
 
----
+## Usage
 
-## Declaring dependencies in Furyfile
+### Prerequisites
 
-Create a _Furyfile_ in the project root with commons and AWS specific dependencies by running
+| Tool                        | Version   | Description                                                                                                                                                    |
+|-----------------------------|-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [furyctl][furyctl-repo]     | `>=0.6.0` | The recommended tool to download and manage KFD modules and their packages. To learn more about `furyctl` read the [official documentation][furyctl-repo].     |
+| [kustomize][kustomize-repo] | `>=3.5.3` | Packages are customized using `kustomize`. To learn how to create your customization layer with `kustomize`, please refer to the [repository][kustomize-repo]. |
 
-> _Furyfile_ is used by _furyctl_ to download dependencies, for more info check [_furyctl's readme_](https://github.com/sighup-io/furyctl)
+### Deployment
```

Comment:
I wrote on the readme: 

```
NB: some packages needs additional configurations, refer to each package documentation for more details.
```

Reply:
It would be nice for the reader if we say at least wich ones need additional steps so he/she does not need to check every readme. But I can live with this a a first version.